test/conftest.py

```diff
@@ -45,15 +45,16 @@ def pytest_addoption(parser):
     parser.addoption('--build-image', '-B', action='store_true')
     parser.addoption('--push-image', '-P', action='store_true')
     parser.addoption('--dockerfile-type', '-T',
-                     choices=['dlc.cpu', 'dlc.gpu', 'dlc.eia', 'pytorch'],
-                     default='pytorch')
+                     # choices=['dlc.cpu', 'dlc.gpu', 'dlc.eia', 'pytorch'],
```

Comment:
Stray comment - can be perhaps removed.

Reply:
We have a cleanup step planned in a subsequent PR where we will remove unnecessary files and code. I will remove these comments at that stage.